R/helper-junitmock.R

```diff
@@ -0,0 +1,26 @@
+# Fix components of JunitReporter that otherwise vary from run-to-run
```

Comment:
Oh I just noticed this in `R/` not `tests/testthat`. Can you please move there? (And then we're done 🎉)